src/lib/scalars/index.ts

```diff
@@ -0,0 +1,25 @@
+import * as NexusSchema from '@nexus/schema'
+import { DateTimeResolver, JSONObjectResolver } from 'graphql-scalars'
+import { GraphQLScalarType } from 'graphql'
+
+export type Scalars = Record<string, GraphQLScalarType>
+
+export const builtinScalars = {
+  Date: new GraphQLScalarType({
+    ...DateTimeResolver,
+    name: 'Date',
+  }),
+  Json: new GraphQLScalarType({
+    ...JSONObjectResolver,
+    name: 'Json',
+    description:
+      'The `JSON` scalar type represents JSON objects as specified by [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf).',
+  }),
+} as const
+
+export function mapBuiltinScalarsToNexusSchemaTypes() {
```

Comment:
What about dogfooding our own api? We could run logic within the app constructor.